### ISSUE TITLE
Map - index? 

### ISSUE THREAD

**Issue by DeanKamali** [yosbelms/react-deco#2]
How to specify an index? was hoping to see it defined like a normal map would. 
```
<Map target={[1, 2, 3]} with={(item, index) =>
      <div key={index}>{item}</div>
} />
```

**Comment by yosbelms** [comment]
As the docs says in https://github.com/yosbelms/react-deco#map it depends on the map implementation of your target.

If it is a regular JS array the mapping function receives arguments in the shape of `(currentValue, index, array)`, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map for more info

**Comment by yosbelms** [comment]
Hey @DeanKamali Looks like I didn't understand you issue. This has been fixed in v0.11.0
